WORKSPACE

```diff
@@ -15,3 +15,7 @@ envoy_dependencies()
 load("//bazel:dependency_imports.bzl", "envoy_dependency_imports")
 
 envoy_dependency_imports()
+
+load("//bazel:dev_binding.bzl", "envoy_dev_binding")
```

Comment:
Can this two line rolled in to repositories.bzl?

Reply:
I think it makes sense to have them as an additional opt-in. Most folks working with WORKSPACE won't want them.

Reply:
That's the point of having LLVM_CONFIG to control in repository impls?

Reply:
As per @jpeach earlier point about not having too much magic, it's probably better to have an explicit statement "yes, I want dev tools" and then the conditioning.

Reply:
A code change (incl. WORKSPACE) to say "yes, I want dev tools" seems too much hassle. Env vars isn't a magic.

Reply:
Given the subjectivity of what is too much hassle, I'm happy to concede on this one, see update.

Reply:
To be clear, "too much hassle" from Envoy consuming project perspective, (i.e. changing WORKSPACE in all of them to get dev tools). I expect these tools will be usable from there.